### PR TITLE
EPL1.0 -> EPL2.0 and contribute.md

### PR DESCRIPTION
**Pull request by svenefftinge**
This PR changes the code license from EPL 1.0 to EPL 2.0 and adds a Contribute.md file.

see https://github.com/eclipse/xtext/issues/1176

### CODE REVIEW

**Comment by cdietrich** [comment]
@svenefftinge looks like you missed have of the header changes

https://www.eclipse.org/projects/handbook/#ip-copyright-headers

see also https://bugs.eclipse.org/bugs/show_bug.cgi?id=530393

**Comment by cdietrich** [comment]
Things to look at

- complete headers
- LICENSE / license.html / epl*html
- about.html / about.properties
- ..... to be checked
- maven metadata / poms (also in gradle)
- feature.xml / feature.properties / license feature in target

**Comment by waynebeaton** [comment]
I really hate to do this, but... while you updated the URLs correctly, you missed the text in at least some of the headers...

image

**Comment by cdietrich** [comment]
@svenefftinge nice. looks much better now. still miss an update to the license feature in the target file

File: org.eclipse.xtend.lib.gwt/about.html [hunk 1]
```diff
@@ -13,8 +13,8 @@ <h3>License</h3>
 
 <p>The Eclipse Foundation makes available all content in this plug-in (&quot;Content&quot;).  Unless otherwise 
 indicated below, the Content is provided to you under the terms and conditions of the
-Eclipse Public License Version 1.0 (&quot;EPL&quot;).  A copy of the EPL is available 
-at <a href="http://www.eclipse.org/legal/epl-v10.html">http://www.eclipse.org/legal/epl-v10.html</a>.
+Eclipse Public License Version 2.0 (&quot;EPL&quot;).  A copy of the EPL is available
```
Comment (cdietrich) [inline org.eclipse.xtend.lib.gwt/about.html]: @svenefftinge i also see a newer date in 
the about.htmls in other projects (e.g. emf/jdt/ltk)
so maybe we should simly copy the whole files over.

**Comment by svenefftinge** [comment]
Yes, I was still on it. But now I hope I have it. Can you review, please?
I'll go through the other PRs once this is good to merge.

**Comment by cdietrich** [comment]
the target file is still not there. or github does not show it
https://github.com/eclipse/xtext-lib/blob/8888f2dc2195ad65565779296bc6ba27031cb1e7/releng/releng-target/xtext-lib.target.target#L10

**Comment by svenefftinge** [comment]
Thanks, I'm updating the target file as well now and replace the whole about files.

**Comment by cdietrich** [comment]
ok will recheck later
@szarnekow @miklossy can you please also have a look

**Pull request merged**